internal/namespaces/mnq/v1beta1/mnq_cli.go

```diff
@@ -0,0 +1,1041 @@
+// This file was automatically generated. DO NOT EDIT.
+// If you have any remark or suggestion do not hesitate to open an issue.
+
+package mnq
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/scaleway/scaleway-cli/v2/internal/core"
+	"github.com/scaleway/scaleway-sdk-go/api/mnq/v1beta1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+// always import dependencies
+var (
+	_ = scw.RegionFrPar
+)
+
+func GetGeneratedCommands() *core.Commands {
+	return core.NewCommands(
+		mnqRoot(),
+		mnqNatsAccounts(),
+		mnqNatsCredentials(),
+		mnqSns(),
+		mnqSnsCredentials(),
+		mnqSqs(),
+		mnqSqsCredentials(),
+		mnqNatsAccountsCreate(),
+		mnqNatsAccountsDelete(),
+		mnqNatsAccountsUpdate(),
+		mnqNatsAccountsGet(),
+		mnqNatsAccountsList(),
+		mnqNatsCredentialsCreate(),
+		mnqNatsCredentialsDelete(),
+		mnqNatsCredentialsGet(),
+		mnqNatsCredentialsList(),
+		mnqSnsActivate(),
+		mnqSnsGet(),
+		mnqSnsDeactivate(),
+		mnqSnsCredentialsCreate(),
+		mnqSnsCredentialsDelete(),
+		mnqSnsCredentialsUpdate(),
+		mnqSnsCredentialsGet(),
+		mnqSnsCredentialsList(),
+		mnqSqsActivate(),
+		mnqSqsGet(),
+		mnqSqsDeactivate(),
+		mnqSqsCredentialsCreate(),
+		mnqSqsCredentialsDelete(),
+		mnqSqsCredentialsUpdate(),
+		mnqSqsCredentialsGet(),
+		mnqSqsCredentialsList(),
+	)
+}
+func mnqRoot() *core.Command {
+	return &core.Command{
+		Short:     `This API allows you to manage Scaleway Messaging and Queueing NATS accounts`,
+		Long:      `Messaging and Queuing NATS API.`,
+		Namespace: "mnq",
+	}
+}
+
+func mnqNatsAccounts() *core.Command {
+	return &core.Command{
+		Short:     `MnQ NATS Accounts commands`,
+		Long:      `MnQ NATS Accounts commands.`,
+		Namespace: "mnq",
+		Resource:  "nats-accounts",
+	}
+}
+
+func mnqNatsCredentials() *core.Command {
+	return &core.Command{
+		Short:     `MnQ NATS Credentials commands`,
+		Long:      `MnQ NATS Credentials commands.`,
+		Namespace: "mnq",
+		Resource:  "nats-credentials",
```

Comment:
Not in favor. I would put everything under a nats resources and then create verbs such as create-credentials under it.